scripts/install-flux.sh

```diff
@@ -19,6 +19,6 @@ then
 fi
 
 helm upgrade flux weaveworks/flux --install --recreate-pods --namespace admin -f ${VALUES} \
-    --set "git.path=k8s/${ENV}/common\,k8s/${ENV}/${CLUSTER_NAME}\,k8s/common",git.label=${ENV},git.email=flux-${ENV}@hmcts.net,git.user="Flux ${ENV}" \
+    --set "git.path=k8s/${ENV}/common\,k8s/${ENV}/${CLUSTER_NAME}/static\,k8s/${ENV}/${CLUSTER_NAME}/static-overlay\,k8s/${ENV}/${CLUSTER_NAME}/automated-overlay\,k8s/common",git.label=${ENV},git.email=flux-${ENV}@hmcts.net,git.user="Flux ${ENV}" \
```

Comment:
I thought we were getting rid of common? and it would be extended in the env folder?

Reply:
We can still have common for common static things . Like https://github.com/hmcts/cnp-flux-config/tree/master/k8s/common like namespace and flex-volume